model/trace/database.yaml

```diff
@@ -26,9 +22,6 @@ groups:
     brief: This group defines the attributes used to perform database client calls.
     attributes:
       - ref: db.query.text
-        requirement_level:
-          recommended: >
-            SHOULD be collected by default only if there is sanitization that excludes sensitive information.
```

Comment:
I feel it's important to preserve.

I understand it's not affecting markdown much, but at least in the yaml we specify this under requirement-level. 

Also we can always do better job changing how markdown is rendered and this may become significant there too.

Reply:
Ok, added it back.

Reply:
if is only required with condition, shouldn't this be using something like
```
 requirement_level:
          conditionally_required: If there is sanitization that excludes sensitive information.
```

Reply:
That'd be a stronger requirement level - it'd mean, if it's sanitized, then it must be captured. Right?

Now we just say, it's recommended, so it may not be captured and I can imagine there will be configs to opt in and out to capture this. So I feel recommended is appropriate here. That was already the case prior to the PR - so there would be no change on that here.

Reply:
The PR would not change the status quo, so I don't want to block it on the req_level discussion, but I agree that

> conditionally_required: If there is sanitization that excludes sensitive information.

would be more precise - let's chat on the SIG call and maybe change it in a follow up